backend/ki/views/api.py

```diff
@@ -165,6 +165,7 @@ def app_config(request):
         'infoPages': info_page_links,
         'role': {
             'isAdmin': request.userinfo.get('admin', False),
+            'adminAvailable': request.userinfo.get('admin_available', False),
```

Comment:
Prefix with `is`, for consistency? --> `isAdminAvailable` and `is_admin_available`?